packages/core/src/integer.ts

```diff
@@ -1100,6 +1100,13 @@ const inSafeRange = Integer.inSafeRange
  */
 const toNumber = Integer.toNumber
 
+/**
+ * Converts a variable to a BigInt
+ * @access public
+ * @param {Mixed} value - The variable to convert
```

Comment:
```suggestion
 * @param {!Integer} value - The variable to convert
```

At least if I understood the code right. What's the purpose of exposing this method in the first place? I only takes Integer and is also bound to Integer. So if the user has an Integer, they can just call `int.valueOf` on it. Or am I missing the point?

Reply:
You're entirely correct.